src/feedback/components/FeedbackForm.vue

```diff
@@ -37,24 +37,62 @@
       {{ anyFirstError }}
     </div>
 
-    <div class="row justify-end q-ma-md q-gutter-sm">
-      <QBtn
-        v-if="!isNew"
-        type="button"
-        :disable="!hasChanged"
-        @click="reset"
-      >
-        {{ $t('BUTTON.RESET') }}
-      </QBtn>
-      <QBtn
-        type="submit"
-        color="secondary"
-        :loading="isPending"
-        :disable="!canSave"
-      >
-        <span v-t="isNew ? 'BUTTON.CREATE' : 'BUTTON.SAVE_CHANGES'" />
-      </QBtn>
+    <div class="row q-ma-md">
+      <div class="col">
+        <QBtn
+          v-if="isNew"
+          type="button"
+          @click="isDismissFeedbackDialogVisible = !isDismissFeedbackDialogVisible"
+        >
+          {{ $t('ACTIVITY_FEEDBACK_DISMISS.BUTTON_LABEL') }}
+        </QBtn>
+      </div>
+
+      <div class="col col-grow">
+        <div class="row justify-end q-gutter-sm">
+          <QBtn
+            v-if="!isNew"
+            type="button"
+            :disable="!hasChanged"
+            @click="reset"
+          >
+            {{ $t('BUTTON.RESET') }}
+          </QBtn>
+          <QBtn
+            type="submit"
+            color="secondary"
+            :loading="isPending"
+            :disable="!canSave"
+          >
+            <span v-t="isNew ? 'BUTTON.CREATE' : 'BUTTON.SAVE_CHANGES'" />
+          </QBtn>
+        </div>
+      </div>
     </div>
+    <CustomDialog v-model="isDismissFeedbackDialogVisible">
+      <template #title>
+        {{ $t('ACTIVITY_FEEDBACK_DISMISS.PROMPT_HEADLINE') }}
+      </template>
+      <template #message>
+        {{ $t('ACTIVITY_FEEDBACK_DISMISS.PROMPT_MESSAGE') }}
+      </template>
+      <template #actions>
+        <QBtn
+          v-close-popup
+          flat
+          color="primary"
+          :label="$t('BUTTON.CANCEL')"
+        />
+        <QBtn
+          v-close-popup
+          flat
+          color="primary"
+          data-autofocus
```

Comment:
We use just plain `autofocus` attribute in most places (a couple of uses of `data-autofocus`), which is best?

I see this is probably copied from me using `data-autofocus` in https://github.com/yunity/karrot-frontend/commit/647b877af#diff-a911f8aaf90e43fc2c4adac22b9d5dfa9070db82e1d75a91c640befb99e96178R84 but I don't remember why :)